Libraries/Lists/FlatList.js

```diff
@@ -126,7 +126,13 @@ type OptionalProps<ItemT> = {|
    * Multiple columns can only be rendered with `horizontal={false}` and will zig-zag like a
    * `flexWrap` layout. Items should all be the same height - masonry layouts are not supported.
```

Comment:
Maybe we should add a note that the default value is 1 is here.

Reply:
Something like this?
```suggestion
   * `flexWrap` layout. Items should all be the same height - masonry layouts are not supported.
   *
   * The default value is 1.
```